### ISSUE TITLE
Add TypeScript definitions for usage with TypeScript

### ISSUE THREAD

**Issue by ezintz** [circuit/circuit-sdk#43]
Hello,

I would like to build an application using TypeScript therefore it would be nice if the SDK would contain the definitions. I was starting once already but it would take me a while. So I thought it would be probably easier for you guys and I will just ask you (:

If you do not want to include them directly within the SDK you could contribute them to the community definitions at https://github.com/DefinitelyTyped/DefinitelyTyped

Let me know how you decided and how I may contribute, if I can.

**Comment by rogeru** [comment]
Yes, we intend to add typescript definitions and add them to DefinitelyTyped. There is no planned date yet though.

**Comment by Jonasdero** [comment]
I support this.
